manuscript/ha-docker-swarm/nodes.md

```diff
@@ -63,6 +63,19 @@ Set your local timezone, by running:
 ln -sf /usr/share/zoneinfo/<your timezone> /etc/localtime
 ```
 
+### Install Docker
+
+A fresh installation of Ubuntu does not include Docker, so you will need to install it manually. Run the following command **on each node**
+
+```
+apt install docker.io -y
+```
+
+!!! tip
+    In order not to have to type `sudo` in front of every Docker command in this guide, you can add your user to the `docker` group by executing the following command **on each node**:
```

Comment:
You are missing a security warning that you should not do this in production.

Reply:
Not sure I agree here. If the intended audience is home lab folks, then "production" is subjective. This was taken from the instructions that Digital Ocean published for their users (https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-18-04). I think the risk here is minimal while the benefit of not having to `sudo` everything is more than worth it.

Reply:
Not sure I agree here. If the intended audience is home lab folks, then "production" is subjective. This was taken from the instructions that Digital Ocean published for their users (https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-18-04). I think the risk here is minimal while the benefit of not having to `sudo` everything is more than worth it.

Reply:
Adding a user to the docker group basically gives him root rights. It is always a good idea to inform the end user about this. Also see https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user .

The risk is basically giving every rce on the host root rights.

And to make the docker command more convenient I aliased it to sudo.

Reply:
We'd achieve both convenience and security here by changing the "tip" into a "warning", per https://squidfunk.github.io/mkdocs-material/extensions/admonition/#warning